install-deps.js

```diff
@@ -0,0 +1,37 @@
+const { exec } = require("child_process");
+const { promisify } = require("util");
+const asyncExec = promisify(exec);
+
+const workspaces = [
+  "american-british-translator",
+  "anonymous-message-board",
+  "build-a-pinterest-clone",
+  "chart-the-stock-market",
+  "exercise-tracker",
+  "file-metadata-microservice",
+  "forum-proxy",
+  "image-search-abstraction-layer",
+  "issue-tracker",
+  "manage-a-book-trading-club",
+  "metric-imperial-converter",
+  // "nightlife-coordination-app" - uncomment when this is ported,
+  "p2p-video-chat-application",
+  "personal-library",
+  "request-header-parser-microservice",
+  "secure-real-time-multiplayer-game",
+  "stock-price-checker",
+  "stock-price-checker-proxy",
+  "sudoku-solver",
+  "timestamp-microservice",
+  "twitch-proxy",
+  "url-shortener-microservice",
+  "voting-app",
+  "weather-proxy",
+];
+
+(async () => {
+  for (const workspace of workspaces) {
+    console.log(`Installing: ${workspace}`);
+    await asyncExec(`cd ${workspace} && npm install`);
```

Comment:
```suggestion
    await asyncExec(`cd ${workspace} && npm ci`);
```
I think `ci` makes more sense, so it's reproducible.  We'd need to update https://github.com/freeCodeCamp/demo-projects/blob/master/CONTRIBUTE.md, though.